some updates (#2760)
Azure/azure-sdk-for-cpp: sdk/keyvault/azure-security-keyvault-secrets/test/samples/sample2-backup-restore/sample2-backup-restore.md

## Getting a Secret

> To get a secret from the key vault  you will need to call GetSecret.

Reviewer: ```suggestion
Call GetSecret to retrieve a secret from Key Vault.
```

## Creating a Backup for the secret properties

> In order to get the backup of the secret we need to call BackupSecret, which will return a vector of bytes representing the backed up content.

Reviewer: ```suggestion
Call BackupSecret to retrieve the secret backup.  BackupSecret will will return a vector of bytes representing the backed up content.
```
